src/client/components/Sidebar/ObjectExpertise.js

```diff
@@ -13,9 +13,18 @@ const ObjectExpertise = ({ username, wobject }) => {
   const [experts, setExperts] = useState({ user: {}, users: [] });
   const { users, user } = experts;
   const isUserInTopFive = users.find(u => u.name === username);
+  const [limit, setLimit] = useState(5);
```

Comment:
Are limit and users.length the same value? Do we need limit variable?

Reply:
++